cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java

```diff
@@ -60,19 +63,29 @@
  */
 public final class TokenSecureStoreUpdater implements SecureStoreUpdater {
   private static final Logger LOG = LoggerFactory.getLogger(TokenSecureStoreUpdater.class);
+
   private final YarnConfiguration hConf;
   private final LocationFactory locationFactory;
+  private final SecureStore secureStore;
   private final long updateInterval;
   private final boolean secureExplore;
+  private final boolean isKMSBacked;
 
   @Inject
-  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf, LocationFactory locationFactory) {
+  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf,
```

Comment:
Yes, the constructor can be made package private.

> On Jul 18, 2016, at 3:41 PM, Bhooshan Mogal <notifications@github.com> wrote:
> 
> In cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java <https://github.com/caskdata/cdap/pull/6066#discussion_r71244446>:
> 
> >  
> >    @Inject
> > -  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf, LocationFactory locationFactory) {
> > +  public TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf,
> I meant the constructor. Not the class. There's no usages of the constructor
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/caskdata/cdap/pull/6066/files/e87907bc48cccf7b7eb520b1f5e6ff174ed59113#r71244446>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAFGk-8BxqYvYfHuXmfJA_u07G1iwIr2ks5qXAEJgaJpZM4JOEs0>.
>